k8s/nginx-openresty/container/docker_build_and_push.bash

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+docker build -t bigchaindb/nginx_3scale:3.0 .
+
+docker push bigchaindb/nginx_3scale:3.0
```

Comment:
Let's bump the version to 3.1 to refer to the new repo from now on?

Reply:
Sure, I can do that.